src/main/scala/notifications/Notification.scala

```diff
@@ -0,0 +1,15 @@
+package notifications
+
+sealed trait Notification {
+  val alert: String
+  val user: String
+
+  val title = "Weather Streams Alert"
+
+  override def toString = s"Hi $user!\n\n" +
```

Comment:
```
override def toString =
    s"""
       |Hi $user! 
       |You are receiving this notification because...
       |$alert
       |Best Regards bla bla abl
     """.stripMargin
```

Perhaps you will find this more useful?

Reply:
done